data/presets/shop/deli.json

```diff
@@ -5,12 +5,18 @@
         "area"
     ],
     "terms": [
+        "deli",
+        "delicatessen",
         "lunch",
         "meat",
         "sandwich"
     ],
     "tags": {
         "shop": "deli"
     },
-    "name": "Deli"
+    "name": "Gourmet Food Store",
```

Comment:
I think the idea that this isn’t called a “deli” in North America has been overblown in the OSM community. Not long ago, I rewrote [the `shop=deli` documentation](https://wiki.openstreetmap.org/wiki/Special:PermanentLink/2250424) to distinguish between `shop=deli` as “deli” and `department=store_section` `product=meat;cheese` as “deli counter”.

Ordinary sandwich shops do often have “deli” in the name, but that doesn’t make “deli” the word for them as far as I know. If you want to be sure, replace “Deli” with “Delicatessen”, which sounds fancier.

“Gourmet Food Store” could work too, but “gourmet food” has been coopted so much by the food industry over the years that it might even be more ambiguous than “Specialty Food Store” – a term we should definitely work into this preset somehow.

Reply:
Should "Specialty Food Store" be the primary name or an alias?

Reply:
> I think the idea that this isn’t called a “deli” in North America has been overblown in the OSM community.

I think it's less that this kind of store _isn't_ called a deli (some are, some aren't) and more that there are many sandwich places that _are_ called delis, but are more accurately tagged as `amenity=fast_food` or `amenity=cafe` in the OSM dialect of English.

Reply:
I'd say "Specialty Food Store" would be a good alias (synonym).  I suppose there might be some specialty food stores that don't fit as a `shop=deli`.  I'm not really sure if the tag is supposed to only be used for a specific kind, or for any specialty food store.  Like would a store that specializes in fancy honeys, jams, and crackers only qualify?  How about one that only sells olive oil?

Reply:
I'd say: yes. The bar is not so high for that

Reply:
As another data point, here are the index entries related to delicatessens in [NAICS 2022](https://wiki.openstreetmap.org/wiki/NAICS/2022):

* [333998 All Other Miscellaneous General Purpose Machinery Manufacturing](https://www.census.gov/naics/?input=333998&year=2022&details=333998)
  * Delicatessen scales manufacturing
  * Retail scales (e.g., butcher, delicatessen, produce) manufacturing
* [445110 Supermarkets and Other Grocery Retailers (except Convenience Retailers)](https://www.census.gov/naics/?input=445110&year=2022&details=445110)
  * Delicatessens primarily retailing a range of grocery items and meats
* [445240 Meat Retailers](https://www.census.gov/naics/?input=445240&year=2022&details=445240)
  * Delicatessens (except grocery store, restaurants)
* [722513 Limited-Service Restaurants](https://www.census.gov/naics/?input=722513&year=2022&details=722513)
  * Delicatessen restaurants

NAICS official status across North America, with only minor variation among countries. It doesn’t necessarily dictate how people describe businesses day to day, but it does rationalize business categories with widely understood terminology and concise definitions. (Something for the OSM Wiki to look up to!)

Reply:
> Should "Specialty Food Store" be the primary name or an alias?

I think “deli” comes closer than either “gourmet food store” or “specialty food store”. If we’re thinking about it this hard, then the latter two should be terms but not a preset name.

European connoisseurs may be aghast at how deli meat and deli sandwiches and cold sandwiches in general have become synonymous with delicatessens in the U.S. But what would they make of shops whose taglines include “[gourmet cookies](https://www.bluechipcookiesdirect.com/)”, “[gourmet hot dogs and burgers](https://www.hillbillyhotdogs.com/)”, and “[gourmet liquor](http://www.rossmoor-csd.org/Shopping-Village/Rossmoor-Gourmet-Liquor-and-Deli)”? These are all likely to be tagged `shop=deli` based on the changes here. Ironically, the [Jewish delis](https://en.wikipedia.org/wiki/Jewish_deli) that brought the concept of a deli to the U.S. from Germany wouldn’t count as `shop=deli` because of their renown for sandwiches.

On the other hand, there’s a whole array of non-European ethnic food stores that wouldn’t be described as “delis” but would meet some definition of `shop=deli` because of their focus on fine foods of some sort. For example, there used to be a handmade tofu shop in my neighborhood that was tagged as `shop=deli` `product=tofu`.

Reply:
The point is is that `shop=deli` is a **shop** according to the wiki definition. You can usually not buy take-out food there as much as you usually can't buy take-out food in a grocery store.

Reply:
> You can usually not buy take-out food there as much as you usually can't buy take-out food in a grocery store.

This is one of the sources of confusion. Most grocery stores in the U.S. do prominently sell takeout food. It’s even what makes some grocery stores a destination, like Amazon’s Whole Foods Market chain or any Hispanic grocery store of any size.

“Deli shop” is a less common term that kind of gets the point across, but if the point is to distance it from a sandwich **shop**, then that’s still confusing to someone who hasn’t read up on the wiki’s bright line between a shop and an amenity.

Reply:
To me it seems like the shortened term "deli" is more likely to be inpreted broadly to include strictly take-out or sit-down restaurant delis (that aren't stores at all) and the deli counters included within larger grocery stores.  The full term "delicatessen", on the other hand seems more specific.  Maybe this is just my interpretation, but if not, I'd say using the longhand term as the preset would be an improvement:
```
"name": "Delicatessen",
"aliases": [
    "Fine Food Store",
    "Gourmet Food Store",
    "Specialty Food Store"
],
"terms": [
    "deli",
    "delicatessen",
    "lunch",
    "meat",
    "sandwich"
]
```

Reply:
Ok, made "Delicatessen" primary name and added "deli" in terms of sandwich fast food